### PR TITLE
feat(postcard route/controller): create and test

### PR DESCRIPTION
**Pull request by jesscxu**
What: created the routes and the controllers to create and submit a postcard

Why: to create a postcard

### CODE REVIEW

File: app/create/controller.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+module.exports = function ($scope, Postcard) {
+
+  $scope.loading = false;
+  $scope.error = null;
+  $scope.postcard = null;
+
+  $scope.create = function (payload) {
+
+    $scope.payload = { mail_type: 'usps_first_class' };
```
Comment (itsachen) [inline app/create/controller.js]: You may want to define this outside the scope of `$scope.create`. `views/create.html` refers to `payload` so it should be initialized along with the other scope variables (`loading`, `error`)

File: app/create/index.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+var Angular = require('angular');
```
Comment (itsachen) [inline app/create/index.js:3]: Nit: Newline between 3rd party libraries and internal modules

File: app/create/index.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+var Controller = require('./controller');
+var State = require('./state');
```
Comment (itsachen) [inline app/create/index.js]: Nit: Align `=`'s

**Review by itsachen — CHANGES_REQUESTED** [review]
Just a minor issue and some nits

File: app/create/controller.js (hunk 1 above)
Comment (jesscxu) [reply app/create/controller.js]: But shouldn't the feature flag for payload only be set when the postcard is being created?

**Review by robinjoseph08 — APPROVED** [review]
LGTM!

**Pull request merged**